static/js/lib/api.js

```diff
@@ -243,15 +244,23 @@ export function addCourseEnrollment(courseId: string) {
 }
 
 export function getCoupons(): Promise<Coupons> {
-  return fetchJSONWithCSRF('/api/v0/coupons/');
+  return fetchJSONWithCSRF('/api/v0/coupons/').then(coupons => {
+    if ( !coupons ) {
```

Comment:
What is this for? `coupons` should always return a list of something, which is always truthy in javascript